src/daft-catalog/src/identifier.rs

```diff
@@ -0,0 +1,76 @@
+use std::fmt::Display;
+
+use crate::errors::{Error, Result};
+
+/// An object's namespace (location).
+pub type Namespace = Vec<String>;
```

Comment:
does this need to be `pub`

Reply:
I believe so since Namespace is a pub field of Identifier